src/hotspot/os/linux/cgroupV2Subsystem_linux.cpp

```diff
@@ -101,18 +101,18 @@ char* CgroupV2Subsystem::cpu_quota_val() {
   return os::strdup(quota);
 }
 
-char * CgroupV2Subsystem::cpu_cpuset_memory_nodes() {
-  GET_CONTAINER_INFO_CPTR(cptr, _unified, "/cpuset.mems",
-                     "cpuset.mems is: %s", "%1023s", mems, 1024);
-  return os::strdup(mems);
-}
-
 int CgroupV2Subsystem::cpu_period() {
   GET_CONTAINER_INFO(int, _unified, "/cpu.max",
                      "CPU Period is: %d", "%*s %d", period);
   return period;
 }
 
+char * CgroupV2Subsystem::cpu_cpuset_memory_nodes() {
+  GET_CONTAINER_INFO_CPTR(cptr, _unified, "/cpuset.mems",
+                     "cpuset.mems is: %s", "%1023s", mems, 1024);
+  return os::strdup(mems);
+}
+
```

Comment:
This puts `cpu_period` and `cpu_quota_val` next to each other, as they both read from `cpu.max` in opposite order (`"%*s %d"` and `"%1023s %*d"`). I think it improves readability, as it's not obvious why `%*d` is there otherwise. Still remove?

Reply:
Yes please, it's got nothing to do with this change. Feel free to move it in separate RFE.